test/role_test.js

```diff
@@ -138,6 +139,52 @@ helper.secrets.mockSuite(helper.suiteName(__filename), ['app', 'azure'], functio
     assert(roles.some(role => role.roleId === 'thing-id:' + clientId));
   });
 
+  test('listRoleIds', async () => {
+    await helper.Roles.modify((roles) => roles.splice(0));
+
+    // Create 4 dummy roles
+    await helper.apiClient.createRole('thing-id:' + clientId, {
+      description: 'test role',
+      scopes: ['dummy-scope-1', 'auth:create-role:*', 'dummy-scope-2'],
+    });
+    for (let i=0;i<3;i++) {
+      let tempClientId = clientId + i;
+      await helper.apiClient.createRole('thing-id:' + tempClientId, {
+        description: 'test role',
+        scopes: ['dummy-scope-1', 'auth:create-role:*', 'dummy-scope-2'],
+      });
+    }
+
+    let result = await helper.apiClient.listRoleIds();
+
+    assert(result.roleIds.some(roleId => roleId === 'thing-id:' + clientId));
+    assert(result.roleIds.length === 4);
+  });
+
+  test('listRoleIds (limit, [continuationToken])', async () => {
+    let hashids = new Hashids();
+    let roleIds = [];
+    let query = {limit:1};
+    let allRoleIds = {};
+    let count=0;
+
+    allRoleIds = await helper.apiClient.listRoleIds();
+
+    while (true) {
+      let result = await helper.apiClient.listRoleIds(query);
+      assume(result.roleIds.length).to.be.lessThan(2);
+      query.continuationToken = hashids.decode(result.continuationToken)[0];
+      roleIds = roleIds.concat(result.roleIds);
+      count++;
+      if (!query.continuationToken) {
+        break;
+      }
+    }
+
+    assume(roleIds.sort()).to.deeply.equal(allRoleIds.roleIds.sort());
+    assume(count).to.be.greaterThan(0);
```

Comment:
This will always be true since `count++` will always be executed at least once before reaching the `break` statement. Maybe we should do `greaterThan(1)`?